packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -16,16 +16,17 @@
 
 @import "@material/ripple/common";
 @import "@material/ripple/mixins";
+@import "../mixins";
 
 .mdc-chip {
   @include mdc-ripple-surface;
   @include mdc-ripple-radius-bounded;
   @include mdc-states;
+  @include mdc-chip-border-radius;
+  @include mdc-chip-fill-color;
+  @include mdc-chip-ink-color;
```

Comment:
We should update the invocation of `mdc-states` to pass the chip ink color to it for consistency.

Reply:
I changed `mdc-chip-ink-color` to automatically update the ripple color (the client can always override it).